scipy/stats/tests/test_qmc.py

```diff
@@ -534,6 +555,58 @@ def test_sample_stratified(self):
         assert np.any(sample - expected > 0.5 / n)
 
 
+class TestOptimalLatinHypercube(QMCEngineTests):
+    qmce = partial(qmc.OptimalLatinHypercube, n_perturbations=100)
+    can_scramble = False
+    unscramble_nd = np.array([[0.60881992, 0.00416027],
+                              [0.82837347, 0.26284543],
+                              [0.20290629, 0.83297228],
+                              [0.48412877, 0.89496811],
+                              [0.06857794, 0.43212172],
+                              [0.98461223, 0.5690004],
+                              [0.3424405, 0.14251516],
+                              [0.64745145, 0.70599331]])
+
+    def test_continuing(self, *args):
+        pytest.skip("Not applicable: not a sequence.")
+
+    def test_fast_forward(self, *args):
+        pytest.skip("Not applicable: not a sequence.")
+
```

Comment:
If it has the method, I think there should be a test. Same with `reset`.

Reply:
What about if for cases like this (LHS and this one), I return a warning saying that it's not a sequence and that there is no correlation/link between the previous and next calls?

Reply:
Well, you could, but it would seem even more to me that something is wrong, either with `QMCEngine` insisting on having a `fast_forward` method or with this class being a subclass of `QMCEngine`.

Setting that question aside, you could pretend that the `fast_forward` was useful and override the test with something like this:
```
    def test_fast_forward(self, *args):
        lhs = qmc.LatinHypercube(d=2, seed=95123684)
        lhs.fast_forward(10)
        sample1 = lhs.random(10)

        lhs.reset()
        lhs.random(10)
        sample2 = lhs.random(10)
        assert_allclose(sample1, sample2)
```